src/module/rules/rule-element/grant-item/base.ts

```diff
@@ -42,6 +45,20 @@ class GrantItemRuleElement extends RuleElementPF2e {
                 : {};
     }
 
+    static ON_DELETE_ACTIONS = ["cascade", "detach", "restrict"] as const;
+
+    #getOnDeleteActions(data: GrantItemSource): Partial<OnDeleteActions> | null {
```

Comment:
Does Pick<OnDeleteActions, "granter" | "grantee"> work here?

Reply:
That would be equivalent to `OnDeleteActions`